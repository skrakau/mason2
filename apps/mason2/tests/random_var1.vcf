##fileformat=VCFv4.1
##source=mason_variator
##reference=random.fasta
##INFO=<ID=END,Number=1,Type=Integer,Description="End position of the variant described in this record">
##INFO=<ID=SVLEN,Number=.,Type=Integer,Description="Difference in length between REF and ALT alleles">
##INFO=<ID=SVTYPE,Number=1,Type=String,Description="Type of structural variant">
##INFO=<ID=TARGETPOS,Number=1,Type=String,Description="Target position for duplications.">
##ALT=<ID=INV,Description="Inversion">
##ALT=<ID=DUP,Description="Duplication">
##contig=<ID=1,length=10000>
##contig=<ID=2,length=1000>
#CHROM	POS	ID	REF	ALT	QUAL	FILTER	INFO	FORMAT	simulated
1	76	.	C	G	.	PASS	.	.	1|1
1	256	.	G	<INV>	.	PASS	SVTYPE=INV;END=331;SVLEN=76	.	0|1
1	454	.	C	C]1:525]	.	PASS	SVTYPE=BND	.	1|0
1	455	.	A	[1:599[A	.	PASS	SVTYPE=BND	.	1|0
1	524	.	A	A]1:600]	.	PASS	SVTYPE=BND	.	1|0
1	525	.	A	[1:454[A	.	PASS	SVTYPE=BND	.	1|0
1	599	.	T	T]1:524]	.	PASS	SVTYPE=BND	.	1|0
1	600	.	T	[1:455[T	.	PASS	SVTYPE=BND	.	1|0
1	642	.	C	CGTATTTACAAGATTTTCCGGAAGCTGCCGCTCAACAGACAGCACACATTGGGGGAATAAGGCCTCGATTATCTGGCCC	.	PASS	SVTYPE=INS;SVLEN=78	.	0|1
1	1033	.	T	TCT	.	PASS	.	.	0|1
1	1139	.	T	<DUP>	.	PASS	SVTYPE=DUP;SVLEN=93;END=1231;TARGETPOS=1:1302	.	0|1
1	1506	.	C	<INV>	.	PASS	SVTYPE=INV;END=1571;SVLEN=66	.	0|1
1	1625	.	C	<DUP>	.	PASS	SVTYPE=DUP;SVLEN=92;END=1716;TARGETPOS=1:1812	.	1|0
1	1893	.	G	GGACCGTGTATTGAACCGCCTAAGACCCAGCTCCACCAGCCCTGCGGATGCAAGACCGGTTC	.	PASS	SVTYPE=INS;SVLEN=61	.	1|0
1	1927	.	C	<DUP>	.	PASS	SVTYPE=DUP;SVLEN=95;END=2021;TARGETPOS=1:2091	.	0|1
1	2203	.	T	T]1:2302]	.	PASS	SVTYPE=BND	.	1|0
1	2204	.	T	[1:2392[T	.	PASS	SVTYPE=BND	.	1|0
1	2301	.	A	A]1:2393]	.	PASS	SVTYPE=BND	.	1|0
1	2302	.	C	[1:2203[C	.	PASS	SVTYPE=BND	.	1|0
1	2392	.	G	G]1:2301]	.	PASS	SVTYPE=BND	.	1|0
1	2393	.	G	[1:2204[G	.	PASS	SVTYPE=BND	.	1|0
1	2405	.	T	A	.	PASS	.	.	1|1
1	2410	.	C	<DUP>	.	PASS	SVTYPE=DUP;SVLEN=60;END=2469;TARGETPOS=1:2541	.	0|1
1	2687	.	T	G	.	PASS	.	.	1|1
1	2900	.	T	TTATACGATGGGTTAGCGTGAATACTTTTATACTCGGTGGAAGGAGTATTTAAACTCGCCATTAAGTCGGAGACTGG	.	PASS	SVTYPE=INS;SVLEN=76	.	1|0
1	2982	.	C	CAGCGAAAGGCCCACGACGTAAAAAGCCTCCGGGGTGGTCGCGGGTCTCCGCTTCATACAAGAGTTAACGGTATTAC	.	PASS	SVTYPE=INS;SVLEN=76	.	0|1
1	3446	.	ATAA	A	.	PASS	.	.	1|0
1	3866	.	T	T]1:3946]	.	PASS	SVTYPE=BND	.	0|1
1	3867	.	T	[1:4028[T	.	PASS	SVTYPE=BND	.	0|1
1	3945	.	C	C]1:4029]	.	PASS	SVTYPE=BND	.	0|1
1	3946	.	C	[1:3866[C	.	PASS	SVTYPE=BND	.	0|1
1	4028	.	C	C]1:3945]	.	PASS	SVTYPE=BND	.	0|1
1	4029	.	A	[1:3867[A	.	PASS	SVTYPE=BND	.	0|1
1	4187	.	ATCCCAAGGCTCGTTAGCTGAGGTCGGGCTCCACCAGAGAGAGTCGAGTTAAACGGGCCAACCTACTCGTCGTCGAGTGGTTTACAAG	A	.	PASS	SVTYPE=DEL;SVLEN=87	.	0|1
1	4374	.	A	G,T	.	PASS	.	.	2|1
1	4400	.	T	<DUP>	.	PASS	SVTYPE=DUP;SVLEN=51;END=4450;TARGETPOS=1:4524	.	0|1
1	4792	.	A	C,T	.	PASS	.	.	1|2
1	4828	.	G	C,T	.	PASS	.	.	1|2
1	4914	.	C	C]1:4993]	.	PASS	SVTYPE=BND	.	1|0
1	4915	.	C	[1:5072[C	.	PASS	SVTYPE=BND	.	1|0
1	4992	.	C	C]1:5073]	.	PASS	SVTYPE=BND	.	1|0
1	4993	.	G	[1:4914[G	.	PASS	SVTYPE=BND	.	1|0
1	5072	.	G	G]1:4992]	.	PASS	SVTYPE=BND	.	1|0
1	5073	.	T	[1:4915[T	.	PASS	SVTYPE=BND	.	1|0
1	5053	.	T	A,G	.	PASS	.	.	1|2
1	5139	.	C	<DUP>	.	PASS	SVTYPE=DUP;SVLEN=66;END=5204;TARGETPOS=1:5287	.	0|1
1	5161	.	C	T	.	PASS	.	.	1|1
1	5189	.	A	C,T	.	PASS	.	.	2|1
1	5456	.	G	<INV>	.	PASS	SVTYPE=INV;END=5528;SVLEN=73	.	0|1
1	5564	.	C	CGACCGTTCCAGCTGGTCGGAATATCCGGAGTTCTTCCACTACTGGATTGTAGTGGTCCCGTCAAACTCACACATTATTTCATCTACTCC	.	PASS	SVTYPE=INS;SVLEN=89	.	1|0
1	5744	.	A	<DUP>	.	PASS	SVTYPE=DUP;SVLEN=82;END=5825;TARGETPOS=1:5888	.	1|0
1	5937	.	A	AGAAATGTTACCTCGGGAGAGAAGACCTTTTGTACCGTTTAGAAGAGTAGACATATCAA	.	PASS	SVTYPE=INS;SVLEN=58	.	1|0
1	6156	.	C	CGCGTCCAGGCTCAGTCGCTCCTTAACTGTCGCCAAATTCCCGAGATCTAAGCAGCA	.	PASS	SVTYPE=INS;SVLEN=56	.	1|0
1	6199	.	A	<INV>	.	PASS	SVTYPE=INV;END=6251;SVLEN=53	.	1|0
1	6383	.	A	<INV>	.	PASS	SVTYPE=INV;END=6472;SVLEN=90	.	0|1
1	6491	.	G	GCAACTGATTATACCTCCAAGGTACTCTCAAGCACACTGTAATGCCTACACGATAAGTCGAGTCGAGCCAGGACGGAGCGATTATAGGA	.	PASS	SVTYPE=INS;SVLEN=88	.	1|0
1	6619	.	G	<INV>	.	PASS	SVTYPE=INV;END=6710;SVLEN=92	.	0|1
1	6929	.	T	<INV>	.	PASS	SVTYPE=INV;END=7024;SVLEN=96	.	1|0
1	7196	.	T	TGTAATTACCCGATCACTATACGCAACTCGCACACACTCCTTTCTAGTGGACAATCCTGATGATGTTGTCGTTATAGAATCGTGTCTAA	.	PASS	SVTYPE=INS;SVLEN=88	.	1|0
1	7312	.	T	TATG	.	PASS	.	.	0|1
1	7552	.	G	<INV>	.	PASS	SVTYPE=INV;END=7608;SVLEN=57	.	0|1
1	7670	.	A	<DUP>	.	PASS	SVTYPE=DUP;SVLEN=76;END=7745;TARGETPOS=1:7823	.	1|0
1	7722	.	CAC	C	.	PASS	.	.	0|1
1	8372	.	A	AAAATAGCATGGTCCACAGAGCTGTATTTGACCATAGCGAGATCGCTTTCTTCTA	.	PASS	SVTYPE=INS;SVLEN=54	.	0|1
1	8425	.	G	<INV>	.	PASS	SVTYPE=INV;END=8518;SVLEN=94	.	1|0
1	8495	.	G	T	.	PASS	.	.	1|1
1	8684	.	A	A]1:8783]	.	PASS	SVTYPE=BND	.	1|0
1	8685	.	G	[1:8847[G	.	PASS	SVTYPE=BND	.	1|0
1	8782	.	C	C]1:8848]	.	PASS	SVTYPE=BND	.	1|0
1	8783	.	G	[1:8684[G	.	PASS	SVTYPE=BND	.	1|0
1	8847	.	A	A]1:8782]	.	PASS	SVTYPE=BND	.	1|0
1	8848	.	T	[1:8685[T	.	PASS	SVTYPE=BND	.	1|0
1	8894	.	G	G]1:8957]	.	PASS	SVTYPE=BND	.	0|1
1	8895	.	T	[1:9020[T	.	PASS	SVTYPE=BND	.	0|1
1	8956	.	G	G]1:9021]	.	PASS	SVTYPE=BND	.	0|1
1	8957	.	T	[1:8894[T	.	PASS	SVTYPE=BND	.	0|1
1	9020	.	T	T]1:8956]	.	PASS	SVTYPE=BND	.	0|1
1	9021	.	T	[1:8895[T	.	PASS	SVTYPE=BND	.	0|1
1	8943	.	T	TCCAAG	.	PASS	.	.	1|0
1	9202	.	A	C,G	.	PASS	.	.	1|2
1	9491	.	G	<INV>	.	PASS	SVTYPE=INV;END=9552;SVLEN=62	.	0|1
1	9720	.	G	T	.	PASS	.	.	1|1
1	9954	.	G	T	.	PASS	.	.	1|0
1	9954	.	G	C	.	PASS	.	.	0|1
1	9968	.	T	TGGGTCTCAGCCGTTTAAGAAGCGCTGTGTTTACCTACCCACGGTGGTTTCCCAAATGACTGACGAGTCCTTCAAT	.	PASS	SVTYPE=INS;SVLEN=75	.	0|1
2	84	.	C	<INV>	.	PASS	SVTYPE=INV;END=146;SVLEN=63	.	1|0
2	250	.	G	<DUP>	.	PASS	SVTYPE=DUP;SVLEN=90;END=339;TARGETPOS=2:414	.	1|0
2	750	.	T	G	.	PASS	.	.	1|0
2	750	.	T	A	.	PASS	.	.	0|1
