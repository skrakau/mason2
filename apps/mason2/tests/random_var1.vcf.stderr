MASON VARIATOR
==============

__OPTIONS_____________________________________________________________________

FASTA IN             	random.fasta
SV SIZE TSV IN       	
VCF OUT              	random_var1.vcf
FASTA OUT            	random_var2.fasta
BREAKPOINT TSV OUT   	random_var1_bp.txt

NUM HAPLOTYPES       	2
HAPLOTYPE SEP        	"/"

SNP RATE             	0.001
SMALL INDEL RATE     	0.001

MIN SMALL INDEL SIZE 	1
MAX SMALL INDEL SIZE 	6

SV INDEL RATE        	0.001
SV INVERSION RATE    	0.001
SV TRANSLOCATION RATE	0.001
SV DUPLICATION RATE  	0.001

MIN SV SIZE          	50
MAX SV SIZE          	100

SIM. METHYL. LEVELS  	1
METHYLATION OUT FILE 	random_var1_meth.fasta
METHYLATION LEVELS
  C   MU             	0.0001
  C   SIGMA          	1e-05
  CG  MU             	0.6
  CG  SIGMA          	0.03
  CHG MU             	0.08
  CHG SIGMA          	0.008
  CHH MU             	0.05
  CHH SIGMA          	0.005


__PREPARATION_________________________________________________________________

Loading Reference Index random.fasta ... OK (2 seqs)

__SIMULATION__________________________________________________________________


Simulation...
  1
  snps:                24
  small indels:        5
  structural variants: 36
  2
  snps:                0
  small indels:        1
  structural variants: 0
OK.


DONE.
